explore/src/main/scala/explore/ExplorePWA.scala

```diff
@@ -119,8 +119,8 @@ object ExplorePWA {
             import scala.concurrent.ExecutionContext.Implicits.global
             // https://vite-plugin-pwa.netlify.app/guide/periodic-sw-updates.html
             val isOffline =
-              !(!(r.installing && !js.isUndefined(org.scalajs.dom.window.navigator))) ||
-                (dom.window.navigator.hasOwnProperty("connection") && dom.window.navigator.onLine)
+              !(!(r.installing && !js.isUndefined(org.scalajs.dom.window.navigator)) ||
```

Comment:
Is this change intended or automated? It seems to change the logic.

Reply:
Yes, it is intended. I posted in Slack about it. The original version was a double `not`. Which negated itself, of course. My belief is that was meant to be what I changed it to because the val is `isOffline` and the it seems like that second clause with `dom.window.navigator.onLine` should be part of the `not`.

Reply:
@cquiroz ?

Reply:
This is the original code
``` 
      if (r.installing || !navigator)
        return

      if (('connection' in navigator) && !navigator.onLine)
        return
```